app/build.gradle

```diff
@@ -65,6 +66,7 @@ dependencies {
     compile 'com.android.support:recyclerview-v7:24.1.1'
     compile 'com.google.android.gms:play-services-gcm:8.4.0'
     compile 'com.google.android.gms:play-services-auth:8.4.0'
+    compile 'com.squareup.picasso:picasso:2.5.2'
```

Comment:
Picasso dependency is already there in the project https://github.com/zulip/zulip-android/blob/master/app/build.gradle#L87
This would give an error of duplicate dependency?